cwms-data-api/src/main/java/cwms/cda/data/dto/texttimeseries/TextTimeSeries.java

```diff
@@ -0,0 +1,275 @@
+package cwms.cda.data.dto.texttimeseries;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.databind.PropertyNamingStrategies;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import com.fasterxml.jackson.databind.annotation.JsonPOJOBuilder;
+import cwms.cda.api.errors.FieldException;
+import cwms.cda.data.dto.CwmsDTO;
+import hec.data.timeSeriesText.DateDateComparator;
+import hec.data.timeSeriesText.DateDateKey;
+import hec.data.timeSeriesText.TextTimeSeriesRow;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.NavigableMap;
+import java.util.Set;
+import java.util.TreeMap;
+
+
+@JsonDeserialize(builder = TextTimeSeries.Builder.class)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonNaming(PropertyNamingStrategies.KebabCaseStrategy.class)
+public class TextTimeSeries extends CwmsDTO {
+//  This Monolith version of this class has:
+// https://bitbucket.hecdev.net/projects/SC/repos/hec-monolith/browse/hec-monolith/src/main/java/hec/data/timeSeriesText/TextTimeSeries.java
+//    private ITimeSeriesDescription _timeSeriesDescription;
+    
+    private final String id;
```

Comment:
rename to `name` to match numerical time series

Reply:
this is where I was wondering if it should use something like the commented out ITimeSeriesDescription.  It would break id/name into its component parts(annoying) but let us specialize the Interval-type.   @adamkorynta  I was hoping you could weigh in on this more.  how should this handle regular, irregular, psuedo-regular etc.

Reply:
We need the full id, the interval offset, and the location time zone. Given the LRTS interval rename migration is underway I don't think we need to do anything further as far as breakdown unless CDA started doing interval processing itself. If the timeseries description included ID, Offset, and Loc Time Zone, that would be fine. Otherwise having three separate variables seems ok to me. 

Maybe the non-graphQL-but-kinda-like-graphql updates down the line would see a catalog endpoint with just those three pieces.